lib/views/pages/main_page.dart

```diff
@@ -10,17 +10,18 @@ class MainPage extends GetView<MainController> {
 
   @override
   Widget build(BuildContext context) => SafeArea(
-        child: Obx(() => Scaffold(
-              key: controller.scaffoldKey,
-              body: Stack(
-                children: [
-                  pageList[controller.nav.value]['view']! as Widget,
-                  CustomAppbar(
-                    title: pageList[controller.nav.value]['name'] as String,
-                  ),
-                  const CustomBottomBar()
-                ],
-              ),
-            )),
-      );
+          child: Scaffold(
+        key: controller.scaffoldKey,
+        body: Stack(
+          children: [
+            Obx(
+              () => pageList[controller.nav.value]['view']! as Widget,
+            ),
+            CustomAppbar(
+              title: pageList[controller.nav.value]['name'] as String,
```

Comment:
This title parameter can be removed since we are already using get.find<> in customAppBar class it takes care of reusing the controller instance.